pjlib/src/pj/sock_bsd.c

```diff
@@ -905,4 +905,127 @@ PJ_DEF(pj_status_t) pj_sock_accept( pj_sock_t serverfd,
 }
 #endif	/* PJ_HAS_TCP */
 
+/*
```

Comment:
First of all, I would suggest putting the whole implementation in `sock_common.c` instead of `sock_bsd.c`, so that it can be used even when the platform doesn't use `sock_bsd.c`.